pkg/server/testdata/test.ini

```diff
@@ -0,0 +1,761 @@
+# possible values : production, development
```

Comment:
I wonder if this is necessary since this file is generated [here](https://github.com/grafana/grafana/pull/26517/files#diff-6fbf282916b625fa025c46f40dd231d2R105)

Reply:
@papagian Oops, no it shouldn't be there. Thanks!